Added ODC Web Console Bug Fixes (#30143)
openshift/openshift-docs: release_notes/ocp-4-7-release-notes.adoc

* With Serverless 0.10, the latest supported versions were updated for `trigger`, `subscription`, `channel`, and `IMC`.  Static models corresponding to each showed an API version of `beta`. The API version for eventing resources is now updated to `v1` and the UI now shows the latest supported version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1890104[*BZ#1890104*])

> * Previously, when the user switched between workloads on the *Monitoring Dashboard* tab, for example from deployment to *All workloads*, the dashboard displayed a white canvas and no chart was displayed. This issue has been fixed now and the dashboard displays charts when the user switches between workloads. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1911129[*BZ#1911129*])

Reviewer: for example, from a specific deployment to *All workloads*,

Author: Done!

> * Previously, when the user switched between workloads on the *Monitoring Dashboard* tab, for example, from a specific deployment to *All workloads*, the dashboard displayed a white canvas and no chart was displayed. This issue has been fixed now and the dashboard displays charts when the user switches between workloads. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1911129[*BZ#1911129*])

Reviewer 2: ```suggestion
* Previously, when the user switched between workloads on the *Monitoring Dashboard* tab, for example, from a specific deployment to *All workloads*, the dashboard displayed a white canvas and no chart. This issue has been fixed; the dashboard now displays charts when the user switches between workloads. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1911129[*BZ#1911129*])
```

> * Previously, the *Monitoring Alert* icon was missing on the workload in the *Topology* view. Currently, the following severity labels are being supported: `critical`, `warning`, `info` and `none`. This issue has been fixed now and anything apart from the above mentioned labels is treated as `warning`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925200[*BZ#1925200*])

Reviewer 3: The Monitoring alert decorator didn't show up because anything apart from the mentioned labels was treated as `info` and we do not show info decorator on the topology node.

How about if we can phrase it like:
```suggestion
* Previously, anything apart from `critical`, `warning`, `info` and `none` was treated as `info`. As a result, the *Monitoring Alert* decorator was not displayed on the workload in the *Topology* view for a severity label like `Critical`. This issue has been fixed now and anything apart from the above mentioned labels is treated as `warning`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925200[*BZ#1925200*])
```

Author: Thanks @debsmita1

Fixed!

> * Previously, anything apart from `critical`, `warning`, `info` and `none` was treated as `info`. As a result, the *Monitoring Alert* decorator was not displayed on the workload in the *Topology* view for a severity label like `Critical`. This issue has been fixed now and anything apart from the above mentioned labels is treated as `warning`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925200[*BZ#1925200*])

Reviewer: This issue is quite confusing:

- s/anything apart from `critical`,/monitoring alerts with levels such as `critical`,
- was treated/were treated
- *Monitoring Alert* decorator/*Monitoring Alert* icon
- Comma after `info` in `info` and `none`
- anything apart from the above mentioned labels is treated as `warning`. - Does this mean - Anything apart from critical, warning , info and none will be considered as warnings instead of info?

Please make sure the case is captured as it is in the GUI. critical/Critical

> * Previously, monitoring alerts with severity levels such as `critical` and `warning` were treated as `info` level alerts. As a result, the *Monitoring Alert* icon was not displayed on the workload in the *Topology* view for these alerts. This issue has been fixed now and alerts like `critical` are treated as `warning` level alerts and a *Monitoring Alert* icon is displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925200[*BZ#1925200*])

Reviewer 2: ```suggestion
* Previously, monitoring alerts with severity levels such as `critical` and `warning` were treated as `info` level alerts. As a result, the *Monitoring Alert* icon was not displayed on the workload in the *Topology* view for these alerts. This issue is now fixed; alerts like `critical` are treated as `warning` level alerts and a *Monitoring Alert* icon is displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925200[*BZ#1925200*])
```
